packages/backend-test-utils/src/database/TestDatabases.ts

```diff
@@ -60,18 +61,19 @@ export class TestDatabases {
     ids?: TestDatabaseId[];
     disableDocker?: boolean;
   }): TestDatabases {
-    const defaultOptions = {
-      ids: Object.keys(allDatabases) as TestDatabaseId[],
-      disableDocker: isDockerDisabledForTests(),
-    };
+    const ids = options?.ids;
+    const disableDocker = options?.disableDocker ?? isDockerDisabledForTests();
 
-    const { ids, disableDocker } = Object.assign(
-      {},
-      defaultOptions,
-      options ?? {},
-    );
+    let testDatabaseIds: TestDatabaseId[];
+    if (ids) {
+      testDatabaseIds = ids;
+    } else if (this.defaultIds) {
+      testDatabaseIds = this.defaultIds;
```

Comment:
```suggestion
    } else if (TestDatabases.defaultIds) {
      testDatabaseIds = TestDatabases.defaultIds;
```

ooh this is a sneaky one - `this` isn't what you think in a static context - i think the old code happens to work "accidentally" since both this method and the setter below are static, but in a regular method you could not access `this.defaultIds`. I like the consistency of doing it on the class name this way; then it works in all contexts.

Reply:
TIL, thanks for catching this one, fixed up 👍